admin/class-boldgrid-backup-admin-archive-details.php

```diff
@@ -131,8 +131,13 @@ public function render_archive() {
 			$this->core->delete_archive_file();
 		}
 
+		if ( ! empty( $_FILES['file'] ) ) {
+			$this->core->upload->upload_archive_file();
+		}
+
 		$this->enqueue_scripts();
 		$this->core->archive_actions->enqueue_scripts();
+		$this->core->auto_rollback->enqueue_home_scripts();
```

Comment:
Is this correct, we're enqueuing auto rollback home scripts on the archive details page?

Reply:
Yes. The enqueue_home_scripts() method is what includes boldgrid-backup/admin/js/boldgrid-backup-admin-home.js in a page.